src/commons/utils/corio_utils.py

```diff
@@ -125,21 +128,34 @@ def run_local_cmd(cmd: str) -> tuple:
         return False, error
 
 
-def create_file(file_name: str, size: int) -> str:
+def create_file(file_name: str, size: int, data_type: str or bytes = bytes) -> str:
     """
-    Create file with random data.
+    Create file with random data(string/bytes), default is bytes.
 
-    :param size: Size in bytes
-    :param file_name: File name
+    :param size: Size in bytes.
+    :param file_name: File name or file path.
+    :param data_type: supported data type string(str)/byte(bytes) while create file.
     """
     base = KIB * KIB
-    file_path = os.path.join(DATA_DIR_PATH, file_name)
+    if os.path.isdir(os.path.split(file_name)[0]):
+        file_path = file_name
```

Comment:
Please explain the need of this if statement.

Reply:
if absolute file path passed then to handle such case this condition introduced.